src/raftstore/store/region_snapshot.rs

```diff
@@ -193,6 +195,7 @@ impl RegionIterator {
             start_key,
             end_key,
             region,
+            panic_when_exceed_bound: true,
```

Comment:
How about moving it to `IterOption` so that there is only one Option?

Reply:
I don't think so, because it is only used in test.